### PR TITLE
[wip] add vagrant image for amazon linux 2 for testing 

### PR DESCRIPTION
**Pull request by sblack4**
I thought I'd draw your guys' attention to this because I think it's worthwhile. Right now I'm using this image pulled from Vagrant's cloud https://app.vagrantup.com/bento/boxes/amazonlinux-2 / https://github.com/chef/bento/blob/master/packer_templates/amazonlinux/amazon-2-x86_64.json

### CODE REVIEW

File: cis/packer.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
   "variables": {
     "region": "us-east-1",
     "instance_type": "t3.medium",
-    "associate_public_ip": "true",
```
Comment (smiller171) [inline cis/packer.json:5]: Why are you defaulting to no public IP for the builder? That can make it a lot more annoying to build from a laptop.

Comment (smiller171) [reply cis/packer.json:5]: @sblack4 You never answered this...

**Comment by sblack4** [comment]
hello AWS Image builder here we come!

**Comment by cdaniluk** [comment]
this PR was just a draft, and we're moving away from packer now so not worth the time to finish this

**Pull request closed without merging**